diy-omd.ino

```diff
@@ -16,38 +20,54 @@ DallasTemperature temperatureSensors(&oneWire);
 elapsedMillis timeMillis;
 elapsedMillis secondaryTimeMillis;
 elapsedMillis tempTimeMillis;
+elapsedMillis testMillis;
 
 const int escPin = 24;
-const int hallEffectSensorPin = 14;
 
 Servo ESC;     // create servo object to control the ESC
-
-const int OMD = 0;
-const int BALANCE = 1;
-const int AXE_550_CALIBRATE = 2;
-const int MODE = OMD;
-
+const int BALANCE = 100;
+const int OMD = 101;
+const int AXE_550_CALIBRATE = 102;
+const int OFF = 103;
+int MODE = OMD;
+const int hallEffectSensorPin = 14;
 const int RC = 0; // rc cars
 const int QUAD = 1; // quadcopters
 const int escType = RC;
-
 const int stopValue = escType == RC ? 90 : 0; // quadcopters don't go in reverse!
 const int reverseValue = 0; // only valid for RC!
 const int OMD_accValue = escType == RC ? 115 : 36;
-const int OMD_goValue = escType == RC ? 123 : 50;
-const int BALANCE_goValue = escType == RC ? 108 : 36; // go slower while we balance!
-const int AXE_550_CALIBRATE_goValue = 180;
-const int goValue = MODE == OMD ? OMD_goValue : (
+int OMD_goValue = escType == RC ? 123 : 50;
+int BALANCE_goValue = escType == RC ? 108 : 36; // go slower while we balance!
+int AXE_550_CALIBRATE_goValue = 180;
+int goValue = MODE == OMD ? OMD_goValue : (
   MODE == BALANCE ? BALANCE_goValue : AXE_550_CALIBRATE_goValue
 );
 volatile int hallEffectCounter = 0;
 int revolutions = 0;
 int numGos = 0;
-const int SWITCH_DELAY = 250;
+int onSequence = 250;
+int offSequence = 250;
+
+SoftwareSerial btSerial(7,8); // RX, TX (from pinout, not BL)
+String inData;
+bool start = false;
```

Comment:
We can make this a local variable in the `processIncomingBTData` function instead. I'd also rename it to `parsingStarted` based on how we use it.